crypto.go

```diff
@@ -120,3 +124,28 @@ func generateNonce() (string, error) {
 
 	return uuid, nil
 }
+
+func decryptToken(encryptedConnectToken string, key *rsa.PrivateKey) (result string, err error) {
+	// token was encoded as a urlsafe base64 so it can be transfered in a url
+	var cipherBytes []byte
+	if cipherBytes, err = urlSafeBase64ToBytes(encryptedConnectToken); err != nil {
+		return
```

Comment:
I believe it's usually preferable to do `return nil, err` than to take advantage of the implicit returns, it makes the code more obvious and easier to figure what what is actually be returned (but this is just my perception of it, not sure if it's correct so take with a grain of salt eheh)

Reply:
That makes sense to me, but that will have to be `return "", err` because you can't use nil for the string return type, right?